spec/models/inventory_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Inventory, type: :model do
+  describe 'Inventory model' do
+    inventory_queries { Inventory.new(name: 'Edward', user_id: 1) }
+  end
```

Comment:
[OPTIONAL] I think that we shouldn't use describe in this context, and preferably use the factory bot rails gem to create an inventory:
let(:inventory) { FactoryBot.create(:inventory, name: 'Edward', user_id: 1) }

Reply:
Hello @guerreiropedr0 , I will make changes on this later in other tasks, thanks for the feedback